test/unit/src/math/Vector3.tests.js

```diff
@@ -303,11 +302,6 @@ export default QUnit.module( 'Maths', () => {
 			assert.strictEqual( a.y, y, 'Identity rotation: check y' );
 			assert.strictEqual( a.z, z, 'Identity rotation: check z' );
 
-			a.applyQuaternion( new Quaternion( x, y, z, w ) );
-			assert.strictEqual( a.x, 108, 'Normal rotation: check x' );
-			assert.strictEqual( a.y, 162, 'Normal rotation: check y' );
-			assert.strictEqual( a.z, 216, 'Normal rotation: check z' );
```

Comment:
Why did you remove these unit tests?

Reply:
These unit tests are silly. They don't actually proof the correctness of the actual rotation. A non-normalized quaternion that is applied to the rotation algorithm can only serve as a kind of fingerprint for the formulas if the arbitrary numbers that get were put in, generate a certain output. As the formulas have changed, they don't provide the same output for non-normalized quaternions. All other unit-tests that work on actual versors work perfectly fine.